src/components/form/Form.tsx

```diff
@@ -1,161 +1,94 @@
 import LabelField from "../inputAndLabel/LabelField.tsx";
-import InputField from "../inputAndLabel/InputField.tsx";
-
-import { Color } from "../colors.ts";
-import {
-  ChangeEvent,
-  KeyboardEvent,
-  SyntheticEvent,
-  useEffect,
-  useState,
-} from "react";
-
+import { useForm } from "react-hook-form";
 import FormButton from "./FormButton.tsx";
-import { nanoid } from "nanoid";
-
+import { Color } from "../colors.ts";
 import { StyledForm } from "./Form.styles.ts";
-
-//retrieve data from local storage
-//if no data is saved return empty object
-const getPlants = () => {
-  const storedPlants = JSON.parse(localStorage.getItem("formData"));
-  if (!storedPlants)
-    return {
-      plants: [
-        {
-          plantID: "",
-          plantName: "",
-          plantImage: "",
-        },
-      ],
-    };
-  return storedPlants;
-};
+import { nanoid } from "nanoid";
+import InputField from "../inputAndLabel/InputField.tsx";
+import NavItem from "../navItem/NavItem.tsx";
 
 const Form = () => {
-  const [initialFormData, setInitialFormData] = useState(getPlants);
-  const [singlePlant, setSinglePlant] = useState(initialFormData.plants[0]);
-
-  //IMAGE PROBABLY WILL BE ADDED IN ANOTHER ITERATION
-  // const [image, setImage] = useState('')
-  //
-  // const onImageChange = (event) => {
-  //     setImage(URL.createObjectURL(event.target.files[0]))
-  // }
+  const storedPlants = JSON.parse(localStorage.getItem("formData"));
```

Comment:
let's use some typescript data model here. This is the source of data for the entire file, we can avoid many mistakes by providing a type here.

Reply:
Thanks, added! I also added this part `JSON.parse(localStorage.getItem("formData") || "")` because of TS error: 
Argument of type 'string | null' is not assignable to parameter of type 'string'.
  Type 'null' is not assignable to type 'string'.
Solution from stackoverflow - https://stackoverflow.com/questions/67700374/use-localstorage-getitem-with-typescript